src/main/java/org/example/DAO/imp/CustomerDao.java

```diff
@@ -0,0 +1,138 @@
+package org.example.DAO.imp;
```

Comment:
Change from `org.example` to your own package naming.

Reply:
Done.